gatsby-node.js

```diff
@@ -83,3 +84,28 @@ exports.createPages = async ({ graphql, actions }) => {
     })
   })
 }
+
+exports.onPostBuild = async ({ graphql }) => {
+  const { data } = await graphql(`
+    {
+      pages: allSitePage(filter: { path: { regex: "/^/en/|^/es//" } }) {
+        nodes {
+          path
+        }
+      }
+    }
+  `)
+
+  return fs.writeFile(
+    path.resolve(__dirname, "scripts/testing/page-paths.json"),
+    JSON.stringify(
+      data.pages.nodes.map((node) => {
+        return {
+          url: node.path,
+        }
+      }),
+      null,
+      2
+    )
+  )
+}
```

Comment:
This is just generating a JSON file that has all the page paths on the site. It happens as part of the production build that runs on `gatsby build`. Not `gatsby develop`. (I think?)

Only including paths that start with en or es. Ones without a language exist, but just redirect to English. Percy only gives you so many screenshots each month, so this helps conserve.